aip/aog/3010.md

```diff
@@ -15,41 +15,41 @@ permalink: /aog/3010
 
 # Actions on Google Vertical Integration Webhook Format
 
-If an action is part of a vertical program that uses Google Fulfillment, it
-usually needs to expose a webhook that Google will call to execute user queries
-via a structured protocol. This AIP defines requirement for the webhook format.
+If an action is part of a vertical program that requires a webhook for Google
+to integrate with the action, the webhookd format **must** follow this AIP.
 
 ## Guidance
 
+### Custom method name
+
+Custom methods in a vertical program are defined by Google and implemented by
+third-party actions. If word separation is required for the method name,
+`lowerCamelCase` **should** be used.
+
+Examples:
+
+- `finalize`
+- `completeTransaction`
+
 ### Base URL in Action Package
 
 The `baseUrl` configured in Action Package defines the common part of the
 actual execution URL of the webhook. The base URL **may** include a trailing
 slash, but **must** work if `:customMethodName` is appended directly. The
-endpoint **must** be HTTPS and **must** have valid certificate.
+endpoint **must** be HTTPS and **must** have a valid certificate.
 
 Examples:
 
-- `https://us-central1.cloudfunctions.net/myWebhook/`
+- `https://us-central1.cloudfunctions.net/myWebhook/` (**not**
+  `https://us-central1.cloudfunctions.net/myWebhook`)
 - `https://test.com/assistant/api`
 
 ### Actual execution URL
 
-The actual execution URL **should** be `baseUrl` appended with
-`:customMethodName`, where `customMethodName` is the Google Fulfillment method
-name.
+Vertical program **must** compute the actual execution URL by appending
+`baseUrl` with `:customMethodName`.
 
 Examples:
 
 - `https://us-central1.cloudfunctions.net/myWebhook/:completeTransaction`
 - `https://test.com/assistant/api:completeTransaction`
-
-### Google Fulfillment method name
```

Comment:
I liked this section. :( What if, instead of removing it, we just remove mentions of GF, e.g. `### Custom method names`?

Reply:
I actually did move it up and renamed this section as "Custom method name".

Reply:
Oops, sorry, I totally missed that!